### ISSUE TITLE
Can't install phpdox on my symfony environnement

### ISSUE THREAD

**Issue by jframos0** [theseer/phpdox#328]
Hi,

I work on Symfony 3.4, with phpUnit 7.0.0 and php-timer 2.0.0. I would try phpdox on my project but i have this problem with composer require.

![image](https://user-images.githubusercontent.com/10958038/37331380-0dd90960-26a4-11e8-94ca-0538c9e9a7d9.png)

**Comment by theseer** [comment]
Hi,

thanks for the bug report.

tl;dr: Tools should not be installed via composer. Use the phar version of phpdox and consider using phive to install it.

Longer version:
Mixing dependencies of the application runtime with development and tool needs is a very bad idea. Mixing the requirements of different tools is inherently worse. The result of this can be seen with the problem you just reported. To make things even worse, there is no easy fix here: To use php-timer 2.0 as phpunit 7 requires, phpdox would need to get updated to also use php-timer 2.0.x. While that itself is a trivial change, php-timer 2 does require PHP 7.1+ while phpDox (currently) still supports php 5.6+. Since the API and namespace changed, there is no trivial change here unless i'd provide a compat layer. A lot of effort to simply get a time ;) and only until phpDox' php requirement will get raised.

Other than with the newest versions of PHPUnit, my goal is to support every PHP-Version that at least still gets security fixes from the PHP Project. So for now, that means PHP 5.6 needs to be supported until the end of 2018. 

Again, to quote my tl;dr, installing phpunit as well as phpdox or any other tool for that matter as their respective phar would make that problem go away and as another benefit would wipe all the crap you don't need for your project at runtime from the vendors folder.
